recipe/patches/disable_16_threads.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/nc_test4/run_par_test.sh.in b/nc_test4/run_par_test.sh.in
+index 2bd12fff..bd132470 100644
+--- a/nc_test4/run_par_test.sh.in
++++ b/nc_test4/run_par_test.sh.in
+@@ -16,9 +16,9 @@ echo "Testing MPI parallel I/O without netCDF..."
+ echo
+ echo "Testing very simple parallel I/O with 4 processors..."
+ @MPIEXEC@ -n 4 ./tst_parallel
+-echo
+-echo "Testing simple parallel I/O with 16 processors..."
+-@MPIEXEC@ -n 16 ./tst_parallel3
++# echo
++# echo "Testing simple parallel I/O with 16 processors..."
++# @MPIEXEC@ -n 16 ./tst_parallel3
```

Comment:
Are we good with disabling this test for all systems?  I think so, since a 16-processor test isn't very meaningful on CI but just want to make sure.

Seems to have done the trick in mpich for ppc64le

Reply:
Let’s just set the number of mpi ranks to 4 but run the test?

Reply:
That patch was [only applied for `ppc64le`](https://github.com/conda-forge/libnetcdf-feedstock/pull/94/files#diff-e178b687b10a71a3348107ae3154e44cR26). I can also try with `-n 4` if you want, but in any case it would be for `ppc64le` only.